src/main/java/org/wso2/lsp4intellij/requests/Timeout.java

```diff
@@ -45,33 +30,9 @@
         Arrays.stream(Timeouts.values()).forEach(t -> timeouts.put(t, t.getDefaultTimeout()));
     }
 
-    public static int CODEACTION_TIMEOUT = timeouts.get(CODEACTION);
-
-    public static int CODELENS_TIMEOUT = timeouts.get(CODELENS);
-
-    public static int COMPLETION_TIMEOUT = timeouts.get(COMPLETION);
-
-    public static int DEFINITION_TIMEOUT = timeouts.get(DEFINITION);
-
-    public static int DOC_HIGHLIGHT_TIMEOUT = timeouts.get(DOC_HIGHLIGHT);
-
-    public static int EXECUTE_COMMAND_TIMEOUT = timeouts.get(EXECUTE_COMMAND);
-
-    public static int FORMATTING_TIMEOUT = timeouts.get(FORMATTING);
-
-    public static int HOVER_TIMEOUT = timeouts.get(HOVER);
-
-    public static int INIT_TIMEOUT = timeouts.get(INIT);
-
-    public static int REFERENCES_TIMEOUT = timeouts.get(REFERENCES);
-
-    public static int SIGNATURE_TIMEOUT = timeouts.get(SIGNATURE);
-
-    public static int SHUTDOWN_TIMEOUT = timeouts.get(SHUTDOWN);
-
-    public static int SYMBOLS_TIMEOUT = timeouts.get(SYMBOLS);
-
-    public static int WILLSAVE_TIMEOUT = timeouts.get(WILLSAVE);
+    public static int getTimeout(Timeouts type) {
+        return timeouts.get(type);
```

Comment:
@NipunaRanasinghe seems like there is a issue still with the timeout API, for example if the consumer tries to override a timeout value by calling the plugins client `setTimeout(Timeouts, int)` API, then we will loose all the default timeout values. So shouldn't we try to put the timeout values to existing map when we call set `setTimeout(Timeouts, int)` ?

Reply:
@gayanper Fixed :) Could you please recheck now?

Reply:
Sure i will try in few hours time and update the issue.

Reply:
@NipunaRanasinghe its working